packages/framework/presence/api-report/presence.alpha.api.md

```diff
@@ -135,29 +135,78 @@ export namespace InternalUtilityTypes {
     };
 }
 
+// @alpha @sealed
+export interface Latest<T> extends LatestCommon<T> {
+    readonly events: Listenable<LatestEvents<T, ProxiedValueAccessor<T>>>;
+    getRemote(attendee: Attendee): LatestData<T, ProxiedValueAccessor<T>>;
+    getRemotes(): IterableIterator<LatestClientData<T, ProxiedValueAccessor<T>>>;
+}
+
+// @alpha
+export function latest<T extends object | null, Key extends string = string>(args: LatestArguments<T> & {
+    validator: StateSchemaValidator<T>;
+}): InternalTypes.ManagerFactory<Key, InternalTypes.ValueRequiredState<T>, Latest<T>>;
+
 // @alpha
 export function latest<T extends object | null, Key extends string = string>(args: LatestArguments<T>): InternalTypes.ManagerFactory<Key, InternalTypes.ValueRequiredState<T>, LatestRaw<T>>;
 
 // @alpha
 export interface LatestArguments<T extends object | null> {
     local: JsonSerializable<T> & JsonDeserialized<T> & (object | null);
     settings?: BroadcastControlSettings | undefined;
+    // (undocumented)
+    validator?: StateSchemaValidator<T> | undefined;
 }
 
 // @alpha @sealed
-export interface LatestClientData<T> extends LatestData<T> {
+export interface LatestClientData<T, TValueAccessor extends ValueAccessor<T>> extends LatestData<T, TValueAccessor> {
     // (undocumented)
     attendee: Attendee;
 }
 
+// @alpha (undocumented)
+export interface LatestCommon<T> {
+    readonly controls: BroadcastControls;
+    getStateAttendees(): Attendee[];
+    get local(): InternalUtilityTypes.FullyReadonly<JsonDeserialized<T>>;
+    set local(value: JsonSerializable<T> & JsonDeserialized<T>);
+    readonly presence: Presence;
+}
```

Comment:
Do we want to factor out the common stuff into a common interface or does that muddy the API too much?